apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -43,6 +62,20 @@ const Table = <T extends object>({
 				<tbody className={styles["table-body"]}>
 					{table.getRowModel().rows.map((row) => (
 						<tr className={styles["table-row"]} key={row.id}>
+							{onRowSelect && (
+								<td className={styles["table-data"]}>
+									<input
+										checked={selectedIds?.has(
+											(row.original as unknown as Identifiable).id,
```

Comment:
we can't be sure it will be id. We need to pass a property like `rowIdKey` - "id" or `getRowId` - a function that accepts a row and returns id.